kernel/memory_x86_64/src/lib.rs

```diff
@@ -182,14 +176,15 @@ pub fn find_section_memory_bounds(boot_info: &impl BootInformation) -> Result<(A
                 data_flags = Some(flags);
                 "nano_core .bss"
             }
-            ".page_table" => {
-                // We bootstrap the page table from the CR3 register.
+            // This appears when compiling for BIOS.
+            ".page_table" | ".stack" => {
                 continue;
             }
-            ".stack" => {
-                stack_start = Some((start_virt_addr, start_phys_addr));
-                stack_end   = Some((end_virt_addr, end_phys_addr));
-                "initial stack"
+            // This appears when compiling for UEFI.
+            ".bootloader-config" => {
+                // TODO: Ideally we'd mark .bootloader-config as not allocated
+                // so the bootloader doesn't load it.
+                continue;
```

Comment:
Please adjust the below debug message to be better/more correct/more specific
```suggestion
                debug!("     no need to map this section, it is only used by the bootloader for config.");
                continue;
```

Reply:
I think that's as correct as it can be. Only the bootloader uses it for config; hence, Theseus doesn't need it.